client/common/common.c

```diff
@@ -533,11 +533,16 @@ menu_with_options(struct client *client)
     if (client->grab) {
         bm_menu_set_filter(menu, "Loading...");
         // bm_menu_grab_keyboard(menu, true);
-        bm_menu_render(menu);
+        if (!bm_menu_render(menu))
+            goto fail;
```

Comment:
Would it be okay to not check bm_menu_render here? It would fail later in the real loop in the end.

Reply:
I don't think I'd be a problem to avoid checking here, as the call isn't in a loop and so it can't cause the 100% CPU issue. And as you say, it'll fail in the next `bm_menu_render` call, and quit (or is it even possible that the Wayland IPC miraculously recovers in the meanwhile?).

The only difference I see is that the bemenu process will stick around for a while reading all the items from stdin. But that's actually what was already happening before (even when the `XKB_KEY_Escape` trick worked). So I've gone ahead and removed this check here.